bin/benchpark

```diff
@@ -283,13 +285,27 @@ def benchpark_setup_handler(args):
 
         run_command(f"{ramble_exe} repo add --scope=site {source_dir}/repo")
 
+    if not initializer_script.exists():
+        with open(initializer_script, "w") as f:
+            f.write(
+                f"""\
+if [ -n "${{_BENCHPARK_INITIALIZED:-}}" ]; then
+    return 0
+fi
+
+. {spack_location}/share/spack/setup-env.sh
+. {ramble_location}/share/ramble/setup-env.sh
+
+export SPACK_DISABLE_LOCAL_CONFIG=1
```

Comment:
@scheibelp Will disabling local config here make Spack run in Verbose mode when called from Ramble?

Reply:
No. Although to clarify, do you want this to happen? Or not to happen? Spack has a few notions of verbosity  (`spack -d`, and apparently also `spack --verbose`, though I don't actually think the latter has much effect and `-d` is typically what we use to get info when something goes wrong).

Reply:
What I think we want is to get the spack -d output when Ramble calls Spack and the build fails.

So, let me clarify my question:

Will setting this env var make Ramble call spack with -d?

Reply:
There is currently not an environment variable that does this in Spack (I can see the appeal though).

Reply:
See https://github.com/LLNL/benchpark/pull/109/files#r1454138774 - it looks like we can generally force running Spack in debug mode (when done via Ramble).